server/internal/testutil/sqlite.go

```diff
@@ -78,5 +79,10 @@ func SqliteDBFromContext(ctx context.Context, t *testing.T) *sqlx.DB {
 		t.Fatal(err)
 	}
 
+	_, err = db.Exec("PRAGMA foreign_keys = ON;")
```

Comment:
Could we use `infrastructure.InitDB` instead to avoid duplication?

Reply:
Unfortunately no. If you try to use `infrastructure.InitDB` you have a circular dependency.
```
package github.com/oxidrive/oxidrive/server/internal/infrastructure/file
	imports github.com/oxidrive/oxidrive/server/internal/testutil
	imports github.com/oxidrive/oxidrive/server/internal/infrastructure
	imports github.com/oxidrive/oxidrive/server/internal/infrastructure/file: import cycle not allowed in test
```